applications/mlflow/chart/mlflow/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: mlflow
+description: A Helm chart for MLflow - Open source platform for the machine learning lifecycle.
+type: application
+version: "0.1.2"
+appVersion: "2.10.0"
+home: https://github.com/mlflow/mlflow/tree/master/charts/mlflow
+sources:
+  - https://github.com/mlflow/mlflow/tree/master/charts/mlflow
+dependencies:
+- name: replicated
+  version: "1.0.0-beta.20"
```

Comment:
Any chance we should use ^ here to automatically pull in new versions? I'm also will to accept that you don't want things changing underneath you. Feels like a dependency that you might want updates to, I'm not sure.